### ISSUE TITLE
snipeitapp.com Header - Missing Hover States for Accessibility

### ISSUE THREAD

**Issue by onehotprocessor** [snipe/snipe-it#103]
Issue:
On the snipeitapp.com webpage, the header is missing hover states (logo), or have hover states that are difficult to note if they are selected (navigation items - turn from light grey to white).

Expected:
Hover states are noted easily. 

URL: http://snipeitapp.com/

Environment:
Mac OSX 10.8.5, Chrome 31.0.1650.57

![screen shot 2013-12-01 at 6 19 23 pm](https://f.cloud.github.com/assets/1160009/1651525/fb131d1a-5ade-11e3-896c-f0da729fcaad.png)
